bin/mutePublicSquadNotifications.ts

```diff
@@ -0,0 +1,68 @@
+import fastq from 'fastq';
+import '../src/config';
+import createOrGetConnection from '../src/db';
+import { Source, SourceMember } from '../src/entity';
+import {
+  NotificationPreferenceStatus,
+  NotificationType,
+  saveNotificationPreference,
+} from '../src/notifications/common';
+
+interface SourceMemberQueue {
+  userId: string;
+  sourceId: string;
+}
+
+const QUEUE_CONCURRENCY = 1;
```

Comment:
Let's increate the concurrency to 100 otherwise it will be very slow just like the previous script

Reply:
Right. Let me update it quickly.